tests/DependencyPassing.elm

```diff
@@ -1,16 +1,16 @@
 module DependencyPassing exposing (..)
 
 import Expect
-import String.Extra
+import Regex
```

Comment:
(Switched to Regex for this test because `elm/regex` was already published, and `elm-community/string-extra` isn't published yet and needs a bit of work to update for 0.19)